Update README.md (#772)
localsend/localsend: README.md

## Getting Started

> To start LocalSend the from source code, follow these steps:

Reviewer: ```suggestion
To start LocalSend from the source code, follow these steps:
```

> To start LocalSend the from source code, follow these steps:

Reviewer 2: Maybe compile instead?

```suggestion
To compile LocalSend from source code, follow these steps:
```

1. Install [Flutter](https://flutter.dev)
2. Clone the LocalSend repository
3. Run `flutter pub get` to download dependencies
4. Run `dart run build_runner build -d` to generate code
5. Run `cd app`
6. Run `flutter run` to start the app

It may be the case that it doesn't work because of a mismatch between the required Flutter version and the installed Flutter version.

> LocalSend uses [fvm](https://fvm.app) to manage the project Flutter version (specified in [.fvm/fvm_config.json](.fvm/fvm_config.json)). After you install it, run `fvm flutter` instead of `flutter`.

Reviewer 2: I don't get why you would change this.

```suggestion
LocalSend uses [fvm](https://fvm.app) to manage the project Flutter version (specified in [.fvm/fvm_config.json](.fvm/fvm_config.json)). After you installed it, run `fvm flutter` instead of `flutter`.
```

## Contributing

> We welcome contributions from anyone interested in helping improve LocalSend. If you'd like to contribute, there are a few ways to get involved:

Reviewer 2: Why?
